src/Controls/src/Core/PublicAPI/net-maccatalyst/PublicAPI.Unshipped.txt

```diff
@@ -345,4 +345,4 @@ virtual Microsoft.Maui.Controls.Handlers.Items2.ItemsViewDelegator2<TItemsView,
 virtual Microsoft.Maui.Controls.Handlers.Items2.ItemsViewHandler2<TItemsView>.UpdateLayout() -> void
 *REMOVED*override Microsoft.Maui.Controls.Handlers.Compatibility.FrameRenderer.SetNeedsLayout() -> void
 *REMOVED*override Microsoft.Maui.Controls.Handlers.Compatibility.FrameRenderer.MovedToWindow() -> void
-override Microsoft.Maui.Controls.Handlers.Compatibility.VisualElementRenderer<TElement>.MovedToWindow() -> void
\ No newline at end of file
+override Microsoft.Maui.Controls.Handlers.Compatibility.VisualElementRenderer<TElement>.MovedToWindow() -> void
```

Comment:
Can revert this unnecessary change?

Reply:
This is not a change. The same code is only appearing as a change in this PR. When I removed the unshipped code of my fix, this appeared in the PR, and I am unable to remove it. @jsuarezruiz